libmscore/style.cpp

```diff
@@ -1323,7 +1326,7 @@ static const StyleType styleTypes[] {
       { Sid::tupletMinDistance,             "tupletMinDistance",             Spatium(0.5)  },
 
       { Sid::autoplaceEnabled,              "autoplaceEnabled",              true },
-
+      { Sid::qualityUpgradeAllowed,         "qualityUpgradeAllowed",         true }
```

Comment:
Is it necessary to make a style option <code>qualityUpgradeAllowed</code>. If it should be a style option, I think we need a kind of version identification since we might need such a option later for other upgrades in the future?

Reply:
3.6 will be the latest release of MU3, in MU4 we'll prepare another solution